lib/view/airway/checklist/intubation_checklist_page.dart

```diff
@@ -25,6 +27,9 @@ class IntubationChecklistPage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    // Analytics set screen name, stays until another screen changes it
+    Analytics().analyticsScreen(Constants.analyticsIntubationChecklistScreen);
```

Comment:
ditto above for all the other calls to Analytics().analyticsScreen()

Reply:
See above ☝️